examples/test_atari.cpp

```diff
@@ -31,26 +31,23 @@ int main(int argc, char** argv) {
     int act_rep = FLAGS_context;
 
     double reward = 0;
-    double reward_per_game = 0;
     double r = 0;
-
+    bool show_screen = false;
     for (int i = 0; i < 100; i++) {
         auto game_over_str =
             GameSimulator::decode_game_over_code(atari->game_over());
+        auto state = atari->get_state(r);
+
+        StatePacket actions;
+        actions.add_buffer_id("action", {util::get_rand_ind(num_actions)});
+        r = atari->take_actions(actions, act_rep, show_screen);
+
         if (game_over_str != "alive" && game_over_str != "lost_life") {
```

Comment:
i think it is better to check game over before taking the first action, as the game maybe in "game over" state at the very beginning.